### ISSUE TITLE
Make a separate component for output

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#26]
Presently, the output component(Skylight) is present in App component, it will be good to move it to a separate component and require that in App.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
